client/my-sites/domains/domain-search/index.jsx

```diff
@@ -444,6 +449,8 @@ export default connect(
 				isSiteOnWooExpress( state, siteId ) ||
 				isSiteOnEcommerce( state, siteId ),
 			isFromMyHome: getCurrentQueryArguments( state )?.from === 'my-home',
+			preferredView: getPreferredEditorView( state, siteId ),
```

Comment:
`getPreferredEditorView` seems to only be used in the context of the site editor. I don't know that it's wrong to use here, but I want to double check if you've looked at other candidates, @chad1008?

What we're looking for here is the value of this setting, right..? cc @arthur791004 

I guess we might be introducing a similar setting for Simple sites with this project pfsHM7-EF-p2

<img width="740" alt="Screenshot 2024-06-26 at 12 42 17" src="https://github.com/Automattic/wp-calypso/assets/1101677/ce0ea3bd-4399-48ff-b350-813117c72263">

Reply:
The value can be retrieved by `getSiteOption( state, site?.ID, 'wpcom_admin_interface' );`
* Classic style - the value equals to `wp-admin`

Reply:
Thanks @fredrikekelund and @arthur791004. I'll update to PR to use this this method instead.

Reply:
Following up on this feedback, I think I may have started work on a free site, and then for whatever reason completed an upgrade, which gave me access to the **Settings > General > Admin Interface** that isn't normally available on free sites.

As @fredrikekelund noted above, that option is coming for simple sites, so I've added the `wpsh` command to enable that setting to the testing instructions above.